location/templates/apartment.html

```diff
@@ -3,7 +3,13 @@
 {% block breadcrumbs %}
 <a href="{% url 'search' %}">Search</a>
 <a href="{% url 'location' pk=apt.location.id %}">{{ apt.location.full_address }}</a>
-<span>{{ apt.suite_num }}</span>
+<span>
+  {% if apt.suite_num %}
+  {{ apt.suite_num }}
+  {% else %}
+  1
```

Comment:
I think showing `1` could be a bit misleading. How about showing other placeholder, for instance `-`, `N/A`, `Unknown suite number`?

Reply:
what about a `#`? Unknown suite number seems like it indicates that something broke, so I don't particularly love that option. A `-` might be tricky to click also.

Reply:
+1 for `-` is tricky to click lol. `#` is also fine for me